app/config.py.example

```diff
@@ -46,6 +46,12 @@ email_conf = ConnectionConfig(
     USE_CREDENTIALS=True,
 )
 
+# register
+DATABASE_CONNECTION_STRING = "sqlite:///calendar.db"
+
+# security
+JWT_KEY = "c06857128217c661de43d746ecdbe9f1dbc3b3685957fab64512456f639b1881"
```

Comment:
Add request to change this token in the `README.md` (add one-liner python code to generate such code using `secrets`. make sure it's at least 32 bytes long to prevent brute-forcing, as we use HS256)

Reply:
This is how i generated the current key. But i don't really understand what do you mean. To add that python 1 line code to the README.MD file? But keep it in config so that the code will still work until final production?

Reply:
You should add an instruction in the README.md about how you generate this token and put it in the `config.py` file.

Every user have to generate his own key, it's not safe to have the publicly advertised key.